parsers/to-flutter/__snapshots__/to-flutter.spec.ts.snap

```diff
@@ -11,16 +11,73 @@ import 'dart:ui';
 class SpecifyMeasurement {
     SpecifyMeasurement._();
 
-    static const baseSpace01 = 8.00;
-    static const baseSpace02 = 16.00;
-    static const baseSpace03 = 24.00;
-    static const baseSpace04 = 32.00;
-    static const baseSpace05 = 64.00;
-    static const baseSpace06 = 96.00;
-    static const baseSpace07 = 128.00;
-    static const baseSpace08 = 160.00;
-    static const baseSpace09 = 192.00;
-    static const baseSpace10 = 224.00;
+    static const double baseSpace01 = 8.00;
+    static const double baseSpace02 = 16.00;
+    static const double baseSpace03 = 24.00;
+    static const double baseSpace04 = 32.00;
+    static const double baseSpace05 = 64.00;
+    static const double baseSpace06 = 96.00;
+    static const double baseSpace07 = 128.00;
+    static const double baseSpace08 = 160.00;
+    static const double baseSpace09 = 192.00;
+    static const double baseSpace10 = 224.00;
+}",
+    },
+  },
+  {
+    "name": "text-styles.dart",
+    "value": {
+      "content": "
+import 'dart:ui';
+
+class SpecifyTextStyle {
+    SpecifyTextStyle._();
+
+    static const TextStyle body = TextStyle(
+      fontFamily: 'Inter',
+      fontSize: 14.00,
+      fontStyle: FontStyle.normal,
+      fontWeight: FontWeight.w500,
+      
+      letterSpacing: 10.00,
+      color: Color(0xFF1E212B),
+    );
+    static const TextStyle bodyWithOpacity = TextStyle(
+      fontFamily: 'Inter',
+      fontSize: 14.00,
+      fontStyle: FontStyle.normal,
+      fontWeight: FontWeight.w500,
+      
+      letterSpacing: 10.00,
+      color: Color(0x4D1E212B),
+    );
+    static const TextStyle code = TextStyle(
+      fontFamily: 'Fira Code',
+      fontSize: 13.00,
+      fontStyle: FontStyle.normal,
+      fontWeight: FontWeight.w500,
+      
+      
+      color: Color(0xFFFF8E05),
+    );
+    static const TextStyle list = TextStyle(
+      fontFamily: 'Roboto',
+      fontSize: 14.00,
+      fontStyle: FontStyle.normal,
+      fontWeight: FontWeight.w400,
+      
+      
+      
+    );
+    static const TextStyle title = TextStyle(
+      fontFamily: 'Inter',
+      fontSize: 32.00,
+      fontStyle: FontStyle.normal,
+      fontWeight: FontWeight.w600,
+      decoration: TextDecoration.underline,
+      
+      color: Color(0xFF1E212B),
+    );
```

Comment:
Generated text styles can have a hole because of optional properties, is that an issue?

Reply:
Yes, the code should be as clean as possible.